src/sentry/api/serializers/models/user.py

```diff
@@ -108,6 +122,13 @@ def serialize(self, obj, attrs, user):
                 'dateVerified': i.last_verified,
             } for i in attrs['identities']]
 
+        if attrs['emails'] is not None:
```

Comment:
Oh, actually it can be, but I'm missing a case that'd be a `KeyError` possibly. I'll fix that, then it's possible this can be `None` correctly if there weren't any UserEmails for this user. I should be able to add a test to prove that too.